### DISCUSSION TITLE
Agregue la nomenclatura que faltaba , de todas las users existentes

### DISCUSSION THREAD

**Discussion by Ubarrionuevo** [pgCatamarca2021fs/aula1-grupoc-team-1c#21] [Announcements]
Añadi la nomenclatura que pedian para enumerar cada users respectivamente (#+N° del ID).

**Comment by lionel1014** [comment]
De una Uli!!! voy creando el Spring 1, así ya decidimos que US pertenecen a la misma... igual desde ya tratare del maquetado
